api/envoy/config/core/v3/grpc_service.proto

```diff
@@ -43,6 +43,12 @@ message GrpcService {
     string authority = 2
         [(validate.rules).string =
              {min_len: 0 max_bytes: 16384 well_known_regex: HTTP_HEADER_VALUE strict: false}];
+
+    // Indicates the retry policy for re-establishing the gRPC channel
```

Comment:
I think this comment is inaccurate.  If I'm reading the code correctly, this actually controls recreating the gRPC *stream*, not the gRPC channel.

Reply:
agreed, will fix